app/views/account/users/edit.html.erb

```diff
@@ -0,0 +1 @@
+<p>Editting user</p>
```

Comment:
REMOVE ALL CODE FROM /account/userss (controller, views, etc)
you have different task